client/my-sites/checkout/cart/test/cart-coupon.js

```diff
@@ -176,10 +176,10 @@ describe( 'cart-coupon', () => {
 					} }
 				/>
 			);
-			applyCoupon.mockReset();
+			removeCoupon.mockReset();
 			component.find( '.cart__remove-link' ).simulate( 'click', event );
-			expect( applyCoupon.mock.calls.length ).toBe( 1 );
-			expect( applyCoupon.mock.calls[ 0 ][ 0 ] ).toBe( '' );
+			expect( removeCoupon.mock.calls.length ).toBe( 1 );
+			expect( removeCoupon.mock.calls[ 0 ][ 0 ] ).toBe( undefined );
```

Comment:
A few alternatives may provide nicer output:

Typically I'd suggest [`.toHaveBeenCalledWith(arg1, arg2, ...)`](https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwitharg1-arg2-). I'm not sure how that will behave with `undefined`. If you want to match something like `toBe( undefined )` or `toBe( null )`, there are special cases: [`.toBeUndefined()`](https://facebook.github.io/jest/docs/en/expect.html#tobeundefined).